napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -155,12 +153,11 @@ def __init__(self, layer):
         self.paint_button = QtModeRadioButton(
             layer, 'paint', Mode.PAINT, tooltip='Paint mode (P)'
         )
-        btn = 'Cmd' if sys.platform == 'darwin' else 'Ctrl'
         self.fill_button = QtModeRadioButton(
-            layer, 'fill', Mode.FILL, tooltip=f'Fill mode ({btn})'
+            layer, 'fill', Mode.FILL, tooltip='Fill mode (F)'
```

Comment:
So it's a little funny here, but we actually allow users to temporarily enable fill and erase using modifier keys, which is why these were documented as they were - see https://github.com/napari/napari/blob/e33c2fd1d9e3456ac7cfef664d25209c411c8da4/napari/layers/labels/_labels_key_bindings.py#L91-L116.

Personally I'm not such a fan of that behaviour as whenever you press the modifier you see the button toggle even if you're just using the modifier as a modifier. I'm curious @jni how you feel about this, and if you've been using this functionality.

I'm not quite sure how to document it either if we do stick with it ...

Reply:
Personally I like the current behaviour, though I haven't developed the muscle memory for it. I can see how it would be very useful if someone masters it, though, so I'd like to keep it, and somehow document it better...

Reply:
thanks for looking!  Just to make sure I'm clear---you want to stick with using 'Control' and 'Alt' for the 2 keys that you can press to temporarily activate fill and erase?  I can certainly change it back.

Reply:
OK, so just re-reading what you both said---  having better documentation around all of this is desired.  I saw that you can do 2 things with each action-- you can either use a key to activate the fill or erase buttons (F or E), or you can temporarily activate the buttons while Control or Alt are pressed.  I saw that both work, but which one would you want in the tooltip?  I used the activated options (F or E) because that was consistent with the rest of the buttons.  However, if temporarily activating a button is getting more use, I can put that in there.  so, now the concern is how to explain in a tooltip?  I can try to explain more in the tooltip...I'll try some options and you can let me know what you think.

Reply:
>  I can try to explain more in the tooltip...I'll try some options and you can let me know what you think.

Thanks! I think (F/E) are more important, but maybe a two line tooltip with the permanent on top, and the temporary below would be good.

```
Fill mode (F)
toggle with (alt)
```

I dunno though, that's just an idea. Curious to see what you come up with!

Reply:
This might make autogeneration really hard, but I think at one point @goanpeca or @Carreau did ask if we'd ever have multiple keys for the same action, and while this isn't quite that it is close!

Reply:
Hum, I hadn't consider the case where keeping it pressed temporarily set it. 
My guess is that can always have an escape hatch, that is to say "F" would be managed by the ActionManager and Alt with custom code/text in tooltip. I'm not too sure it is worth having the complexity (yet), if it is a rare occurence.

Reply:
I update the tooltips.  I liked what you had suggested @sofroniewn and couldn't come up with something better (I tried).